src/main/java/mcmultipart/api/multipart/MultipartHelper.java

```diff
@@ -54,7 +54,7 @@ public static boolean addPart(World world, BlockPos pos, IPartSlot slot, IBlockS
     public static Optional<IPartInfo> getInfo(IBlockAccess world, BlockPos pos, IPartSlot slot) {
         return getContainer(world, pos).map(c -> c.get(slot)).orElseGet(() -> {
             if (world instanceof World) {
-                IBlockState state = world.getBlockState(pos);
+                IBlockState state = world.getBlockState(pos).getActualState(world, pos);
```

Comment:
getActualState is, as far as I know, primarily used for rendering. Are you sure you're using it correctly?